modules/core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/repository/Repository.java

```diff
@@ -84,6 +88,11 @@ public boolean isDefault() {
         return Boolean.TRUE.equals(isDefault);
     }
 
+    /** @since 11.2 */
+    public boolean isHeadless() {
+        return Boolean.TRUE.equals(headless);
+    }
```

Comment:
Why not:
```
    public Boolean isHeadless() {
        return headless;
    }
```
as for the other descriptors?

Reply:
Probably because this one should really return `true` or `false` as it's the object used everywhere :)
(such as in https://github.com/nuxeo/nuxeo-web-ui/pull/973)

Reply:
Yes. I need a single point in which to determine the semantics of a null value. Plan is for it to be false in master (repos are not headless by default) but for it to be true when null and not the default repo in 10.10 so that the repository picker is not suddenly shown.

Reply:
Also, note we do the same with isDefault()